client/my-sites/importer/importer-header.jsx

```diff
@@ -51,31 +51,32 @@ class ImporterHeader extends React.PureComponent {
 
 	controlButtonClicked = () => {
 		const {
-			importerStatus: { importerId, importerState, type },
-			site: { ID: siteId },
-			startImport,
-		} = this.props;
+				importerStatus: { importerId, importerState, type },
+				site: { ID: siteId },
+				startImportFn,
+			} = this.props,
+			tracksType = type.endsWith( 'site-importer' ) ? type + '-wix' : type;
```

Comment:
How are we going to handle tracks if we introduce other site importers?

Reply:
You found my nasty hack! The problem is that in the future when the site parser supports other engines, the user will be using the `site-importer` and the engine will be determined at the point the site is validated. So this would be a `site-importer` event and change to a `site-importer-wix` event later on.

Currently we only have one engine, but that isn't selected for use until the site is validated, but if the user clicks on this in the frontend, then we know it's Wix they're importing from. It's not pretty hardcoding it like this, but I don't see another way at the moment. 

I've just chatted to @bisko about it and I think he reluctantly agrees! He might have some other thoughts on it though.

Reply:
Yes, although the first event will just have `site-importer` as we won't have identified the service used at the time, and then subsequent events will have `site-importer-wix` `site-importer-medium` etc.

At the moment the site importer is almost synonymous with Wix importer, so that's why I've hard-coded it for now.

We could of course change it so the properties are `site-importer` and `site-importer-wix` - with the Wix part coming from the site validation on the server - but there's been some queries as to why the first event doesn't mention Wix.